cmd/influxd/launcher/launcher.go

```diff
@@ -178,51 +172,18 @@ func (m *Launcher) Engine() Engine {
 func (m *Launcher) Shutdown(ctx context.Context) error {
 	var errs []string
 
-	if err := m.httpServer.Shutdown(ctx); err != nil {
-		m.log.Error("Failed to close HTTP server", zap.Error(err))
```

Comment:
Is there any particular reason why you use "Failed to close" in this error message but use "Failed closing" in the others?

Reply:
Not sure why the old code used different forms. The new code is consistent for everything.